handlebars/handlebars.go

```diff
@@ -109,7 +109,10 @@ func (e *Engine) Load() (err error) {
 	defer e.mutex.Unlock()
 	// Set template settings
 	e.Templates = make(map[string]*raymond.Template)
-	raymond.RegisterHelpers(e.funcmap)
+
+	if !e.loaded {
+		raymond.RegisterHelpers(e.funcmap)
```

Comment:
why not use sync.Once ?

Reply:
Good solution! - I will make the changes in a separate PR so my changes above are not part of the branch.